### ISSUE TITLE
FontFaceException: We are missing some required font packages

### ISSUE THREAD

**Issue by ianbrandt** [agentk/gulp-fontgen#17]
Fails to run when installed per the README:

```
$ gulp build-fonts

node_modules/fontfacegen/lib/commands.js:60
            throw new FontFaceException(
            ^
FontFaceException: We are missing some required font packages.
That can be installed with:
brew install fontforge ttf2eot batik-ttf2svg
```

Unfortunately, there does not appear to be a Homebrew formula for batik-ttf2svg:

```
$ brew install fontforge ttf2eot batik-ttf2svg
Error: No available formula with the name "batik-ttf2svg" 
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
Error: No formulae found in taps.
```

**Comment by zakkfleischmann** [comment]
👍 

Having this same issue.

**Comment by zakkfleischmann** [comment]
I was able to figure out a solution that worked for me. First, make sure you've run `brew update`. I then ran each command one-by one. The commands are:

1. `brew install batik` - notice the name difference
2. `brew install ttf2eot`
3. `brew install fontforge` - fontforge had some additional setup commands that needed to be run for me and likely will for you too.

I then ran into an issue with java when trying to run `gulp fonts` that was easily fixed by downloading the latest JDK from Oracle [here](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). 

Hope this helps!

**Comment by sgen** [comment]
Im on Ubuntu so this isnt something I can test.

A Pull request with a solution would be welcome.

**Comment by sgen** [comment]
If either of you are up for it Id like to avoid issues like this in the future by pretesting releases on all OS's
See #20

**Comment by sgen** [comment]
This should be fixed with agentk/fontfacegen#32
